editor/scene_tree_dock.cpp

```diff
@@ -1061,6 +1061,28 @@ void SceneTreeDock::_tool_selected(int p_tool, bool p_confirm_override) {
 				}
 			}
 		} break;
+		case TOOL_TOGGLE_SCENE_UNIQUE_NAME: {
+			List<Node *> selection = editor_selection->get_selected_node_list();
+			List<Node *>::Element *e = selection.front();
+			if (e) {
```

Comment:
You are not using the selection nor element, so the code could be simplified to something like this:
```suggestion
			if (!editor_selection->get_selected_node_list().is_empty()) {
				Node *node = editor_selection->get_selected_node_list().front()->get();
```

Reply:
I copied/pasted the code from the other case values. Did not want it to differ too much from the others.